corehq/apps/reports/standard/deployments.py

```diff
@@ -483,6 +488,40 @@ def _fmt_timestamp(timestamp):
         result[0][1] = table
         return result
 
+    def get_location_column(self, user):
+        assigned_loc_ids = user.get('assigned_location_ids')
+        if not assigned_loc_ids:
+            return '---'
+        primary_loc_id = user.get('location_id')
+        return self._get_formatted_assigned_location_names(primary_loc_id, assigned_loc_ids)
+
+    def _get_formatted_assigned_location_names(self, primary_location_id, assigned_location_ids):
```

Comment:
nit: Can we move this to javascript?

I like it when any rendering operation is done on the browser instead of the server, leading to better response time.

Reply:
We could, although I'll just need to figure out a way to get the assigned/primary location names to the front-end so that the browser can parse them correctly. I'll look into this.

Reply:
Looking through the code, it seems we will have to keep this parsing available on the back-end. This is because when exporting the report ([this](https://github.com/dimagi/commcare-hq/blob/1359e6ed5b12d929d97f5a59f0f0181a3811ccdc/corehq/apps/reports/standard/deployments.py#L463) func), we won't be able to use the code we put on JS.

We could potentially have it in both places, where for report viewing the parsing is done on the front-end and for exports it's done on the back-end, however this feels like we are unnecessarily adding complexity to the code. @mkangia What are your thoughts?